src/platformdirs/android.py

```diff
@@ -56,6 +56,25 @@ def user_log_dir(self) -> str:
             path = os.path.join(path, "log")
         return path
 
+    @property
+    def user_documents_dir(self) -> str:
+        """
+        :return: documents directory tied to the user e.g. ``/storage/emulated/0/Documents``
+        """
+        # Get directories with pyjnius
+        try:
+            from jnius import autoclass  # noqa: SC200
+
+            Environment = autoclass("android.os.Environment")
+            data_dir: str = Environment.getDataDirectory().getAbsolutePath()
+            documents_dir_name: str = Environment.DIRECTORY_DOCUMENTS
+
+            documents_dir = os.path.join(data_dir, "media/0", documents_dir_name)
```

Comment:
You should pass media and 0 separately :thinking: Also how come media is not contained in the e.g. above?

Reply:
I wanted to use a form of the path that people would be more familiar with in the example. `/data/media/0` is the actual location of `/storage/emulated/0`. I'll just replace the example with a media one.